package.json

```diff
@@ -16,93 +16,49 @@
   },
   "license": "AGPL-3.0",
   "scripts": {
-    "prepare": "rimraf openseadragon && git clone https://github.com/openseadragon/openseadragon.git && cd openseadragon && git checkout tags/v2.4.1 && cd .. && npm run fix-openseadragon",
+    "build": "vue-cli-service build",
+    "e2e": "node tests/e2e/runner.js chrome",
     "fix-openseadragon": "replace \"\\}\\(this, function \\(\\) \\{\" \"}(window, function () {\" ./openseadragon/src/openseadragon.js",
-    "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
```

Comment:
The removed command should also be changed in the README - there is stil `npm run dev`